src/App.vue

```diff
@@ -68,11 +71,18 @@
 
         fetch(url, { headers: { Accept: "application/json" } })
```

Comment:
I've told you this before but it still looks like your fetch logic is in your App.vue file. I don't think you're following SOLID principles in App.vue largely due to what's going on around here. Do you think this should still stay here? Let me know what you think

Reply:
We talked about how this project is a UI for the actual API Browser. Ideally, this app only depends on the api-browser and the api-browser depends on fetch. Extracting fetch as Tony says is a first step toward decoupling the api-browser from the UI.

Reply:
I think for now it's ok to leave `fetch` in here. I'm assuming that you are suggesting to move `fetch` out into a JS file, and then just call the wrapper in `request`. There are certain functions that can be extracted and more thoroughly tested (`setLinks`, `handleResponse`). However, there are some side effects that I'm not quite sure how to take care of (setting `App`s instance variables, for example). I'm not sure if it makes sense to return an object from fetch, containing the new values for the instance variables, then using that object in `App` to set its variables...

Reply:
what's wrong with doing what you said? returning an object from a function that calls fetch and setting the instance variables using that?

Reply:
That works. I just wasn't sure if that was the best way to update the instance variables